src/coreservices.cpp

```diff
@@ -422,6 +422,21 @@ void CoreServices::initialize(QApplication* pApp) {
 
     m_pTouchShift = std::make_unique<ControlPushButton>(ConfigKey("[Controls]", "touch_shift"));
 
+    // The following wmainmenubar controls must be created here so that controllers can bind to them
+    // on startup.
+    m_pSkinSettingsControl = std::make_unique<ControlPushButton>(
```

Comment:
This is a binary flag, not a button, right? Shouldn't this be

```suggestion
    m_pSkinSettingsControl = std::make_unique<ControlObject>(
```

?

Reply:
good catch thanks

Reply:
Ping

Reply:
I don't understand the difference between `[Master],skin_settings` and `[Microphone],show_microphone` for example. All those are used for binary pushbuttons.
May someone please elaborate?

Reply:
This makes a difference when linking a midi button and is also used for the default behavior in skins. 
A plain ControlObject has no hints how to link a midi button and should be only used for internal controls.

Reply:
Thank you!

Reply:
so is skin_settings not triggered by midi?  Is that the only reason not to set as a pushbutton?